WordPressAuthenticatorTests/SingIn/AppleAuthenticatorTests.swift

```diff
@@ -0,0 +1,109 @@
+import AuthenticationServices
+@testable import WordPressAuthenticator
+import XCTest
+
+class AppleAuthenticatorTests: XCTestCase {
+
+    // showSignupEpilogue with loginFields.meta.appleUser set will pass SocialService.apple to the delegate
+    func testShowingSignupEpilogueWithApple() throws {
+        WordPressAuthenticator.initializeForTesting()
+        let delegateSpy = WordPressAuthenticatorDelegateSpy()
+        WordPressAuthenticator.shared.delegate = delegateSpy
+
+        // This might be unnecessary because delegateSpy shoudl be deallocated once the test method finished.
```

Comment:
```suggestion
        // This might be unnecessary because delegateSpy should be deallocated once the test method finished.
```

Reply:
Thank you for catching this and its copy!